readthedocs/projects/forms.py

```diff
@@ -529,17 +529,27 @@ class AddonsConfigForm(forms.ModelForm):
 
     class Meta:
         model = AddonsConfig
-        fields = ("enabled", "project")
+        fields = (
+            "enabled",
```

Comment:
I'd say the only thing I noticed is that this field renders as `Enabled`, and should probably be more descriptive regardless of the pattern we use -- `Enable Addons` or something.

Reply:
Yeah, good point. I originally used the layout approach and added some text explaining this was "global". I'll check with the serializer if I can change how it renders since we are already using the `help_text` to explain the same thing.

Reply:
I also thought we can use better labels for:

```
            "external_version_warning_enabled": _("Show a notification on builds from Pull Requests"),
            "stable_latest_version_warning_enabled": _("Show a notification on non-stable and latest versions"),
```

What do you think?

Also also, we can remove the ` enabled` part from all of them and just leave the name of the addon with the checkbox?

Reply:
![Screenshot_2024-01-19_15-43-50](https://github.com/readthedocs/readthedocs.org/assets/244656/e41d84f6-3643-4979-bc04-3ee32eb11fe8)

Reply:
> I also thought we can use better labels for

Yeah those look better! Just a small note that "Pull Request" should use proper noun capitalization, "pull request builds" can be used instead. I follow what GitHub uses: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

> Also also, we can remove the  enabled part 

Aye, we could for this UI. Though thinking ahead to when we split it up to multiple pages/tabs, we might want the full verbose version "Flyout enabled"? I suppose I'd leave the "enabled" part on the text for now and if we want to remove it later we can.